_data/team/roles/releases_and_deployment.yaml

```diff
@@ -0,0 +1,15 @@
+role: Releases and deployment
+tasks: 
+  - Monitoring upstream and downstream package changes
```

Comment:
This is a task I have indeed been doing - but I'm not sure if this is "releases and deployment" necessarily, or just "maintenance of the entire ecosystem".